Dockerfile

```diff
@@ -32,7 +35,6 @@ RUN apk --no-cache --update add \
     libpng \
     make \
     mariadb-client \
-    mariadb-client-libs \
```

Comment:
No longer required?

Reply:
Yeah it has been removed in more recent versions of Alpine but mariadb-client is still there. I do not see any errors while building so I assumed it was working ok.